tooling/src/hypothesistooling/__main__.py

```diff
@@ -368,25 +368,22 @@ def run_tox(task, version):
 
 
 # Via https://github.com/pyenv/pyenv/tree/master/plugins/python-build/share/python-build
-PY27 = "2.7.16"
 PY35 = "3.5.7"
 PY36 = "3.6.9"
 PY37 = "3.7.4"
 PY38 = "3.8.0"
-PYPY2 = "pypy2.7-5.10.0"
 PYPY35 = "pypy3.5-7.0.0"
 PYPY36 = "pypy3.6-7.1.1"
 
 
 @task()
 def install_core():
-    install.python_executable(PY27)
     install.python_executable(PY36)
 
 
-ALIASES = {PYPY2: "pypy", PYPY35: "pypy3", PYPY36: "pypy3"}
+ALIASES = {PYPY35: "pypy3", PYPY36: "pypy3"}
```

Comment:
I assume this is correct, but `pypy3` being both 3.5 and 3.6 looks all kinds of weird.

Reply:
I'll add a comment - this the executable name that we invoke them by :slightly_smiling_face: